### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by kean**
### Description

Fixes #

ℹ Please replace the above with a link to the issue this pull request addresses, as well as a summary of the implementation details.

### Testing Details

ℹ Please replace this with a clear and concise description of the steps required to validate this pull request.

---

- [ ] Please check here if your pull request includes additional test coverage.
- [ ] I have considered if this change warrants release notes and have added them to the appropriate section in the `CHANGELOG.md` if necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ _None._
 
 -->
 
-## Unreleased
+## 12.0.0
```
Comment (crazytonyli) [inline CHANGELOG.md:33]: A note to me and reviewers: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/706 bumped the version but didn't updated the changelog file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ When releasing a new version:
 
 ### Breaking Changes
 
-_None._
+- Update `AtomicWebServerLogEntry` keys [#711]
```
Comment (crazytonyli) [inline CHANGELOG.md:17]: This is added in the wrong place. The "When releasing a new version:" part above has instructions of how to update this file.

**Review by crazytonyli — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (kean) [reply CHANGELOG.md:17]: Since I'm not releasing a new version (yet), I think it's in the right place?

Comment (crazytonyli) [reply CHANGELOG.md:17]: This part is a template inside a comment `<!-- ... -->`, [it's not visible in the rendered html](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/2f70f10e34e22abf56fc6ac4d5d6664c9f69d52f/CHANGELOG.md). You'll need to copy this "Unreleased" section and put it outside of the comment block.

**Comment by mokagio** [comment]
@kean I incorporated this PR in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/718. Thank you for following up.

Closing.

**Pull request closed without merging**